src/sweeps/config/schema.json

```diff
@@ -698,6 +698,11 @@
           "description": "At every eta^n steps, hyperband continues running the top 1/eta runs and stops all other runs",
           "exclusiveMinimum": 1,
           "default": 3
+        },
+        "strict": {
+          "type": "boolean",
+          "description": "Use a more aggresive condition for termination, stops more runs",
+          "default": false
```

Comment:
probably don't need an explicit default here

Reply:
actually lets keep this, but make it default to true

Reply:
nit, indentation